qsignature/src/org/qcmg/sig/Options.java

```diff
@@ -116,6 +116,7 @@
 		parser.accepts("excludeVcfsFile", "file containing a list of vcf files to ignore in the comparison").withRequiredArg().ofType(String.class)
 				.describedAs("excludeVcfsFile");
 		parser.accepts("validation", VALIDATION_STRINGENCY_OPTION_DESCRIPTION).withRequiredArg().ofType(String.class);
+		parser.accepts("stream", VALIDATION_STRINGENCY_OPTION_DESCRIPTION);
```

Comment:
what is the stream option description? will it be the same as "validation"?

Reply:
nope - I have added a new description for the stream option